common/install_android.sh

```diff
@@ -104,3 +104,83 @@ export PATH="${GRADLE_HOME}/bin/:${PATH}"
 echo "PATH:${PATH}"
 
 gradle --version
+
+rm -rf /tmp/gradledeps
+
+_gradledeps=/tmp/gradledeps
```

Comment:
Why put this in /tmp?

Reply:
I planned to remove it eventually, as deps should be stored in `~/.gradle/caches`